src/toil/test/wdl/wdltoil_test.py

```diff
@@ -824,6 +824,21 @@ def test_gs_uri(self):
                 assert os.path.exists(result["ga4ghMd5.value"])
                 assert os.path.basename(result["ga4ghMd5.value"]) == "md5sum.txt"
 
+    def test_check(self):
+        """Test that Toil's lint check works"""
+        wdl = os.path.abspath("src/toil/test/wdl/lint_error.wdl")
+
+        out = subprocess.check_output(
+            self.base_command + [wdl, "-o", self.output_dir, "--logInfo"], stderr=subprocess.STDOUT)
+
+        assert b'UnnecessaryQuantifier' in out
+
+        p = subprocess.Popen(
+            self.base_command + [wdl, "--strict=True", "--logCritical"], stderr=subprocess.PIPE)
+        stderr = p.stderr.read()
+        p.wait()
+        assert p.returncode == 2
+        assert b'Workflow did not pass linting in strict mode' in stderr
```

Comment:
mr-c wants us to use the new way of getting test data that works even after we package Toil.

```suggestion
        with get_data("test/wdl/lint_error.wdl") as wdl:
            out = subprocess.check_output(
                self.base_command + [wdl, "-o", self.output_dir, "--logInfo"], stderr=subprocess.STDOUT)

            assert b'UnnecessaryQuantifier' in out

            p = subprocess.Popen(
                self.base_command + [wdl, "--strict=True", "--logCritical"], stderr=subprocess.PIPE)
            stderr = p.stderr.read()
            p.wait()
            assert p.returncode == 2
            assert b'Workflow did not pass linting in strict mode' in stderr
```